packages/server/src/modules/cart/cart.resolver.ts

```diff
@@ -149,7 +149,10 @@ export class CartResolver {
 
       book = bookDetails;
 
-      if (bookDetails.requests.length === 0) {
+      if (
+        bookDetails.requests.filter(({ deletedAt }) => deletedAt === null)
+          .length === 0
+      ) {
```

Comment:
![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/6266078/49e8b6ca-a705-46ff-8868-bd7949350135)

```suggestion
      if (bookDetails.requests.length === 0) {
```

Reply:
Lol you are right, going to accept your suggestion yep